doc/development.mdwn

```diff
@@ -0,0 +1,172 @@
+Developing LinkChecker
+======================
+
+The following steps describe how to compile LinkChecker from source
+on various platforms.
+
+This is a technical document, if you are looking for ways to
+participate in the community, you should rather look into
+[contributing](contributing).
+
+Requirements
+------------
+On Mac OS X systems, using MacPorts, Fink or homebrew for software
+installation is recommended.
+
+- Install Python >= 2.7.2 from http://www.python.org/
```

Comment:
Seems a rather low version. I think you need at least 2.7.6 for proper TLS support (e.g. SNI)?